aws/resource_aws_eip_test.go

```diff
@@ -763,10 +763,6 @@ resource "aws_eip" "test" {
 
 func testAccAWSEIPConfig_tags_Ec2Classic(rName, testName string) string {
 	return fmt.Sprintf(`
-provider "aws" {
```

Comment:
Since its not a subject that will be dealt with with any future testing, will migrate this to the standardized EC2-Classic handling (https://github.com/hashicorp/terraform-provider-aws/pull/15803) post-merge. It was missed before because it wasn't using `os.Setenv()` like the others.